cmd/swapcli/main.go

```diff
@@ -290,6 +290,34 @@ func cliApp() *cli.App {
 					swapdPortFlag,
 				},
 			},
+			{
+				Name: "claim",
+				Usage: "manually call claim() in the contract for a given swap. " +
```

Comment:
We might need an FAQ item in our Wiki with an example of a use case of where it was successfully used.  It is calling claim outside the automated protocol, so I guess you could say it is a "manual" claim.

Reply:
yeah we should emphasize these should definitely not be used normally, it's mostly for us when we find bugs testing or in the case a user runs into some unexpected and unrecoverable error